public/locales/en/settings/customization/access.json

```diff
@@ -0,0 +1,6 @@
+{
+  "allowAnonymous": {
+    "label": "Allow anonymous",
```

Comment:
Change to guests. Anonymous users are users that have an account with no username.